src/Heatmap/HeatmapSeries/HeatmapSeries.tsx

```diff
@@ -171,9 +85,7 @@ export const HeatmapSeries: FC<Partial<HeatmapSeriesProps>> = ({
   }) => {
     const x = xScale(row.key);
     const y = yScale(cell.x);
-    const fill = fillValueScale(cell.value);
-    const stroke = strokeValueScale(cell.value);
-    const glowFilter = glowValueScale(cell.value);
+    const colorSchemeStyles = getColorSchemeStyles(cell.value, valueScales);
```

Comment:
We need to be specific with our deconstruction of these - don't want just anything being spread out.